### ISSUE TITLE
--date-format '%d/%m/%Y' not working

### ISSUE THREAD

**Issue by kiteloopdesign** [cantino/reckon#111]
I have a CSV with data as follows:

```
F. VALOR,CATEGORIA,SUBCATEGORIA,DESCRIPCION,COMENTARIO,IMPORTE
15/05/2021,Vehiculo y transporte,Parking y garaje,Pago en SERVEIS AMBIENTALS DE CASCASTELLDEFELSES,,-2.1
14/05/2021,Vehiculo y transporte,Transporte publico,Pago en ACCIONA MOTOSHARING ALCOBENDAS ES,,-3.46
14/05/2021,Ocio y viajes,Ocio y viajes (otros),Pago en BICING BARCELONA ES,,-0.35
14/05/2021,Ocio y viajes,Cafeterias y restaurantes,Pago en SHAGANS SL BARCELONA ES,,-35
```

This is day/month/year

I can run reckon with no date modifier, it kind of picks it up automatically, but the output format is on %Y/%m/%d. If I use %d/%m/%Y, I get the following error:

```
I'm having trouble parsing  with the desired format: %d/%m/%Y
```

**Comment by benprew** [comment]
Hmm.... Looks like --date-format is only used when parsing input, and
doesn't affect the output format.  To support the output format, we'll need
to add another option to do that.

Thanks

**Comment by DurhamSmith** [comment]
I am having a similar issue however the issue is with correctly parsing input with this date format (I dont mind if output  is ymd).

**Comment by benprew** [comment]
This is fixed in reckon-0.8.0
